solutions/gatekeeper-policies/guardrails/09-network-security-services/tests/cloudsql-test.yaml

```diff
@@ -0,0 +1,29 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+  apiVersion: sql.cnrm.cloud.google.com/v1beta1
+  kind: SQLInstance
+  metadata:
+    name: sqlinstance-sample-private-ip
+    annotations:
+      config.kubernetes.io/local-config: 'true'      
+  spec:
+    databaseVersion: MYSQL_5_7
+    region: us-central1
```

Comment:
I know it's only a test, but should we point the SQL instance to a region in CA?

Reply:
I'm not familiar with the test of a policy. @lucstjean-ssc can you run a quick test with nane1 and let me know if it works, I will update it

Reply:
The constraint is not actually testing the region configuration and nothing is being deployed. I still you prefer seeing `northamerica-northease1`  as the region.

Reply:
updated with commit https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit/pull/230/commits/aaa7cb66d0764d567443a2cee5a47436c29d1f65